docs/docs-content/vm-management/create-manage-vm/enable-cpu-hotplug.md

```diff
@@ -1,24 +1,59 @@
 ---
-sidebar_label: "Enable CPU Hotplug"
-title: "Enable CPU Hotplug"
-description: "Learn how to enable the CPU Hotplug feature in your VMs using Palette Virtual Machine Orchestrator."
+sidebar_label: "Enable CPU and Memory Hotplug"
+title: "Enable CPU and Memory Hotplug"
+description:
+  "Learn how to enable the CPU and Memory Hotplug feature in your VMs using Palette Virtual Machine Orchestrator."
 sidebar_position: 50
 tags: ["vmo", "cpu-hotplug"]
 ---
 
-You can enable the [KubeVirt CPU Hotplug](https://kubevirt.io/user-guide/compute/cpu_hotplug/) feature in Palette, which
-allows the Virtual Machine (VM) to add and remove virtual CPUs while the VM is running.
+You can enable the [KubeVirt CPU Hotplug](https://kubevirt.io/user-guide/compute/cpu_hotplug/) and
+[KubeVirt Memoruy Hotplug](https://kubevirt.io/user-guide/compute/memory_hotplug/) feature in Palette, which allows the
+Virtual Machine (VM) to add and remove virtual CPUs, and increase or descrease memory, while the VM is active
 
 ## Limitations
 
-- CPU hotplug is not currently supported by the ARM64 architecture.
-- The current CPU hotplug implementation requires the live migration of the VM workload.
+- CPU and memory hotplug is not currently supported by the ARM64 architecture.
+- The current [CPU](https://kubevirt.io/user-guide/compute/cpu_hotplug/#limitations) and
+  [memory](https://kubevirt.io/user-guide/compute/memory_hotplug/#limitations) hotplug implementation requires the live
+  migration of the VM workload.
```

Comment:
The live migration will be done automatically by VMO: how can we mention that?

Reply:
Yes sir :)